src/views/get-started/get-started.js

```diff
@@ -32,7 +32,7 @@ const Faq = () => {
     {
       src: `get-started/monitoring.png`,
       url: `https://mayadata.io`,
-      text: `Unlock the Additional Features and Free Monitoring for OpenEBS`,
+      text: `Get Commercial Support and add DR & other features with Kubera`,
```

Comment:
Change this to: `Get Commercial Support and add DR & other features with products like Kubera`. 

Change the link from "https://mayadata.io" to /support page.